pkg/config/config.go

```diff
@@ -242,11 +242,15 @@ func fillOpenShiftRouteSuffixIfNecessary(nonCachedClient client.Client, configMa
 func ConfigMapPredicates(mgr manager.Manager) predicate.Predicate {
 	return predicate.Funcs{
 		UpdateFunc: func(evt event.UpdateEvent) bool {
-			updateConfigMap(mgr.GetClient(), evt.ObjectNew)
+			if evt.ObjectNew.GetName() == "devworkspace-controller-configmap" {
```

Comment:
please, let's use it from constant

Reply:
Yeah, accidentally left the "lets see if this works" implementation here. Updated to match name/namespace from `ConfigMapReference` (which can be overridden via env var)